package.json

```diff
@@ -37,7 +37,7 @@
     "lint:js": "eslint \"**/*.js\"",
     "lint:yml": "eslint \"**/*.{yml,yaml}\"",
     "lint:styles": "stylelint \"**/*.scss\"",
-    "lint:lit": "yarn lit-analyzer \"src/components/**/*.ts\"",
```

Comment:
could we remove the react deps now?

Reply:
and jsx deps

Reply:
I think yes.
We should also convert files from tsx to ts